Temporarily disable unbuildable packages in extra feed

rajendra-desai-ni opened the pull request:
It seems that many recipes declared as RDEPENDS in the extra packagefeed have dropped from the layers. That doesn't fail the build, but it renders the whole feed metapackage as unbuildable and invalidates even those recipes which would otherwise be buildable.

This change is to unblock building of packagefeed-ni-extra group so that we can debug and fix the disabled packages and then reenable back.

[AB#2703160](https://dev.azure.com/ni/DevCentral/_workitems/edit/2703160/)

**Testing**
- [X] bitbake --continue packagefeed-ni-extra

amstewart commented on files/passwd:
```diff
@@ -57,3 +57,4 @@ opensaf:x:342::::
 # free space
 radiusd:x:301::::
 postgres:x:28::::
+arpwatch:x:402::::
```
Could you move this line up so that the UID/GIDs are sorted in reverse order?

rajendra-desai-ni replied:
I have updated the commit summary line and have also moved arpwatch UID/GIDs up in a proper order.

amstewart reviewed (changes requested):
When you're making individual recipe changes, it's helpful to prepend your commit summary line with the name of the recipe or subsystem that you're changing. ie. `pkggrp-ni-extra: disable unbuildable packages`.

Could you rewrite your summaries in that style?

chaitu236 commented on files/group:
```diff
@@ -70,3 +70,4 @@ tracing:x:341:
 radiusd:x:301:
 postgres:x:28:
 mail:x:8:
+arpwatch:x:402:
```
Where did you get these values from?

rajendra-desai-ni replied:
I just reserved an available static ID to arpwatch without which the extra feeds build would fail.

chaitu236 replied:
Not sure if this is the right thing to do but can merge this for now. We'll have to take a closer look at how best to fix this. Please create a WI for it.

amstewart replied:
It's not unusual that we have to define a uid/gid for a service that is getting pulled into the feeds. Because we chose to use the OE static_uidgid configuration, we have to define all users and groups at build time. The numbers we choose to assign them are arbitrary and only follow the numbering range policy that we defined, and is in comments at the top of the file. FWIW, nothing about his change seems odd to me.

rajendra-desai-ni committed:
packagefeed-ni-extra: temporarily disable unbuildable packages

This change is to unblock building of packagefeed-ni-extra group.

Signed-off-by: Rajendra Desai <rajendra.desai@ni.com>

rajendra-desai-ni committed:
passwd/group: add uid/gid for arpwatch

extra packagefeed fails to build if we do not define
the static ID for arpwatch package.

Signed-off-by: Rajendra Desai <rajendra.desai@ni.com>
